web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -494,163 +667,92 @@ const TeamReviews = ({ periodId }) => {
   return (
     <Root>
       <div className={classes.headerContainer}>
-        <Typography variant="h4">Team Reviews</Typography>
-        {!selectedTeamMember && (
+        <Typography variant="h4">{period?.name ?? ''} Team Reviews</Typography>
+
+        {period && isAdmin && (
+          <div>
+            <Tooltip
+              title={
+                period.reviewStatus === ReviewStatus.OPEN
+                  ? 'Archive'
+                  : 'Unarchive'
+              }
+            >
+              <IconButton
+                onClick={toggleReviewPeriod}
+                aria-label={
+                  period.reviewStatus === ReviewStatus.OPEN
+                    ? 'Archive'
+                    : 'Unarchive'
+                }
+              >
+                {period.reviewStatus === ReviewStatus.OPEN ? (
+                  <Archive />
+                ) : (
+                  <Unarchive />
+                )}
+              </IconButton>
+            </Tooltip>
+            <Tooltip title="Delete">
+              <IconButton
+                onClick={confirmDelete}
+                edge="end"
+                aria-label="Delete"
+              >
+                <Delete />
+              </IconButton>
+            </Tooltip>
+          </div>
+        )}
+        {!selectedMember && (
           <FormControlLabel
             control={
               <Switch checked={includeAll} onChange={toggleIncludeAll} />
             }
             label="Show All"
           />
         )}
-        {selectedTeamMember && (
+        {selectedMember && (
           <Button
             onClick={handleOpenNewRequest}
             className={classes.actionButtons}
-            endIcon={<AddCircleIcon />}
+            endIcon={<AddCircle />}
             variant="contained"
             color="primary"
           >
             Request Review
           </Button>
         )}
       </div>
-      {!selectedTeamMember && loadedReviews.current && (
```

Comment:
We no longer want to display the team members associated with the review period here. Users must click the "+" icon to open the `MemberSelectorDialog` in order to see them.